INFO:root:1693440
INFO:root:No: 1	GFLOPS: 0.00/0.00	result: MeasureResult(runs=(TVMError('[11:39:09] src/schedule/graph.cc:179: Check failed: !visited.count(s.get()) Find loop in compute_at attach group\n\nStack trace returned 10 entries:\n[bt] (0) /home/jian/tvm/lib/libtvm.so(dmlc::StackTrace[abi:cxx11]()+0x5a) [0x7f25dc541e5a]\n[bt] (1) /home/jian/tvm/lib/libtvm.so(dmlc::LogMessageFatal::~LogMessageFatal()+0x28) [0x7f25dc542a48]\n[bt] (2) /home/jian/tvm/lib/libtvm.so(tvm::schedule::CreateAttachPath(tvm::Schedule)+0x21a) [0x7f25dc67f13a]\n[bt] (3) /home/jian/tvm/lib/libtvm.so(tvm::schedule::InferBound(tvm::Schedule const&)+0xcf5) [0x7f25dc6a16c5]\n[bt] (4) /home/jian/tvm/lib/libtvm.so(+0x8aeb58) [0x7f25dc6afb58]\n[bt] (5) /home/jian/tvm/lib/libtvm.so(TVMFuncCall+0x5e) [0x7f25dc919c0e]\n[bt] (6) /usr/lib/x86_64-linux-gnu/libffi.so.6(ffi_call_unix64+0x4c) [0x7f25f5395e40]\n[bt] (7) /usr/lib/x86_64-linux-gnu/libffi.so.6(ffi_call+0x2eb) [0x7f25f53958ab]\n[bt] (8) /usr/lib/python2.7/lib-dynload/_ctypes.x86_64-linux-gnu.so(_ctypes_callproc+0x48f) [0x7f25f55a53df]\n[bt] (9) /usr/lib/python2.7/lib-dynload/_ctypes.x86_64-linux-gnu.so(+0x11d82) [0x7f25f55a9d82]\n\n',),), all_cost=1000000000.0, error_no=1, timestamp=1528828749.326223)	[('tile_n', [8, 1]), ('tile_f', [32, 2]), ('tile_y', [112, 2]), ('reorder', (3, 5, 2, 1, 4, 6, 0))],None,None,906224
INFO:root:1693440
